### PR TITLE
data README first draft

### PR DESCRIPTION
**Pull request by lauriemerrell**
# Description

This PR creates a README summarizing how to access Ghost Buses data in the public S3 bucket. 

## Type of change

- [ ] Bug fix
- [ ] New functionality
- [x] Documentation

## How has this been tested?
Sharing with the group, asking folks to attempt to access data based on this

### CODE REVIEW

File: data_analysis/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* **schedule_summaries**: To compare the realtime data with the schedule, [`static_gtfs_analysis.ipynb`](https://github.com/chihacknight/chn-ghost-buses/blob/main/data_analysis/static_gtfs_analysis.ipynb) ingests a GTFS schedule file and outputs a file that aggregates to a count of unique trips per hour on a given route. There are two flavors:
+    * `route_level`: 
+        * These files do not distinguish between different route directions/destinations (so Howard-bound #22 Clark bus trips are combined with Harrison-bound #22 Clark bus trips). Their schema is:
+            * `date`: Schedule date (can be joined with `bus_hourly_summaries.data_date`)
+            * `route_id`: Route identifier (can be joined with `bus_hourly_summaries.rt`)
+            * `hour`: Schedule hour (can be joined with `bus_hourly_summaries.data_hour`)
+            * `trip_count`: Count of unique trip IDs scheduled to have any scheduled stops within the given hour on the given route (can be compared with `bus_hourly_summaries.trip_count`)
```
Comment (porouspaper) [inline data_analysis/README.md:44]: do trip id's count unique "trips" which are when a bus completes a route in one direction?

File: data_analysis/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+The high-level data flow for the CHN Ghost Buses project is as follows:
+
+```mermaid
+flowchart LR;
+scraper[scrape_data.py scrapes raw realtime bus location data every 5 minutes] --writes data to--> private[JSON files in chn-ghost-buses-private bucket] -.manually batch processed into.-> public[CSV files in chn-ghost-buses-public bucket]
+```
+
+There are a few types of data available in the public S3 bucket:
+
+* **Daily summaries of the scraped realtime data**: To make the raw API response data more user-friendly for analysis, the [`rt_daily_aggregations`](https://github.com/chihacknight/chn-ghost-buses/blob/main/data_analysis/rt_daily_aggregations.ipynb) notebook aggregates the JSON files into three separate aggregated files:
```
Comment (dcjohnson24) [inline data_analysis/README.md:14]: I would add a caveat about the `rt_daily_aggregations` as discussed in #6.

**Review by dcjohnson24 — COMMENTED** [review]
Looks good overall! All the files are accessible from those URLs. I would add that for `pd.read_csv` to work with the `s3` URL, one needs to `pip install s3fs` as well.

 I would also give a few examples of how to access the data without pandas as you showed during the meeting. To use `wget` for example, one could run `wget https://chn-ghost-buses-public.s3.amazonaws.com/bus_full_day_errors_v2/2022-05-20.csv`. Or from the browser, one could drop the URL into the address bar and get an automatic download. 

It would be good to point out that one should replace 'private' with 'public' in the bucket paths in the notebooks. There was a Slack post you wrote on August 18th that had good information about this. Also, running the notebooks will throw errors when trying to write files to both the public and private buckets. One will have to comment out the lines saving to `s3` or run the scripts.

File: data_analysis/README.md (hunk 1 above)
Comment (lauriemerrell) [reply data_analysis/README.md:44]: Trip IDs *should* correspond to scheduled GTFS information: https://gtfs.org/schedule/reference/#tripstxt. So broadly, yes, but there can be some nuances / quirks

**Review by dcjohnson24 — APPROVED** [review]

File: data_analysis/README.md [hunk 3]
```diff
@@ -1,66 +1,41 @@
...
+    * **Errors**: 
+        * These are CSV files that contain all the error data we received from the API, concatenated together for a full day. The schema of the data is exactly what is returned from the API, with only a `scrape_file` field (see above) added that records the name of the S3 resource where the original JSON response that contained this row is saved; see [the `getvehicles` section of the CTA documentation](https://www.transitchicago.com/assets/1/6/cta_Bus_Tracker_API_Developer_Guide_and_Documentation_20160929.pdf) for field definitions from the API.
+        * These files are generated daily between 10 and 11am Central for the prior. There is one file per full day from `2022-05-20` until the day before you are making the request. So, if you are checking on `2022-10-02` after 11am Central, errors will be available up to and including `2022-10-01`.
```
Comment (dcjohnson24) [inline data_analysis/README.md:39]: Add the word "day" after "prior" to "These files are generated daily between 10 and 11am Central for the prior."

**Review by dcjohnson24 — APPROVED** [review]
Thanks for this. The changes look good.

File: data_analysis/README.md (hunk 3 above)
Comment (lauriemerrell) [reply data_analysis/README.md]: Ah good catch, fixed

**Pull request merged**